WebService/v1/Controllers/DeviceGroupController.cs

```diff
@@ -31,18 +31,21 @@ public DeviceGroupController(IStorage storage)
         }
 
         [HttpPost]
+        [Authorize("CreateDeviceGroups")]
```

Comment:
Need constants for these.

Reply:
they are only used once, so I think it is ok